arangod/Replication2/Methods.cpp

```diff
@@ -283,15 +283,10 @@ struct ReplicatedLogMethodsCoordinator final
     auto dbservers = clusterInfo.getCurrentDBServers();
     std::size_t expectedNumberOfServers =
         std::min(dbservers.size(), std::size_t{3});
-    if (options.config.has_value()) {
-      expectedNumberOfServers = options.config->replicationFactor;
-    } else if (!options.servers.empty()) {
-      expectedNumberOfServers = options.servers.size();
-    }
+    expectedNumberOfServers = options.servers.size();
```

Comment:
Maybe we want to keep this feature of just specifying a number of servers for a replicated log or a prototype state by having a separate struct for the `CreateOptions` that has an optional `replicationFactor` in there?

Reply:
Maybe, but that is then a different user story